index.html

```diff
@@ -217,12 +219,12 @@ <h3 id="contact">CONTACT US</h3>
         <section class="form-container" id="desktop-form" >
             <h3 id="contact1">Let’s chat!</h3>
             <P>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </P>
-            <form action="https://formspree.io/f/xdoylybq" method="POST">
+            <form action="https://formspree.io/f/xdoylybq" method="POST" id="desktopForm">
                 <div class="form-box">
                     <input class="input1" type="text" id="firstName" name="name" placeholder="First name" required>
-                    <input class="input2" type="text" id="lasNname" name="name" placeholder="Last name" required>
-                </div>
+                    <input class="input2" type="text" id="lasNname" name="name" placeholder="Last name" required>                </div>
                 <input type="email" id="mail1" name="_replyto" placeholder="jeremy@gmail.com"  maxlength="500" required>
+                <small id="lowerCase1" class="mailErorr formShow">Please write the email in lower case</small>
```

Comment:
Let's show the error message near the submit button.

![image](https://user-images.githubusercontent.com/31897434/128582488-0f6cf693-e8b4-4b88-b3c6-e29b9c9c91cf.png)